vanilla-tests/src/test/java/fixtures/httpinfrastructure/HttpClientFailureTests.java

```diff
@@ -31,14 +31,14 @@ public static Stream<Arguments> httpClientFailureSupplier() {
         return Stream.of(
             Arguments.of(createExecution(() -> client.getHttpClientFailures().head400()), 400),
             Arguments.of(createExecution(() -> client.getHttpClientFailures().get400()), 400),
-            Arguments.of(createExecution(() -> client.getHttpClientFailures().options400()), 400),
+//            Arguments.of(createExecution(() -> client.getHttpClientFailures().options400()), 400),
```

Comment:
commented this because testserver does not implement options calls on /http/failure/client/ cases